projects/bloomreach-experience-ng-sdk/src/lib/common-sdk/utils/page-model.ts

```diff
@@ -52,9 +50,8 @@ export function _buildApiUrl(apiUrls: ApiUrls, preview: boolean, urlPath: string
     url = addComponentRenderingURL(url, componentId, apiUrls);
   }
 
-  const [, ssoHandshake = ''] = (query && query.match(SSO_HANDSHAKE)) || [];
-  if (ssoHandshake) {
-    url += (url.indexOf('?') === -1 ? '?' : '&') + ssoHandshake;
+  if (query) {
+    url += (url.indexOf('?') === -1 ? '?' : '&') + query;
```

Comment:
I personally prefer `!url.includes('?')`over `url.indexOf('?') === -1`

Reply:
Good notice, fixed that. Thanks 👍